### DISCUSSION TITLE
[Feature] Map markers for albums

### DISCUSSION THREAD

**Discussion by WatcherWhale** [immich-app/immich#3418] [Feature Request]
### The feature

It would be nice to show only markers of photos/videos from a certain (shared) album. This way it is easy to visualize only vacation pictures, like the road you took on a road trip.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by andreap-** [comment]
a mapview of the single album would be wellcome when viewing an holiday trip album. I find if usefull to have it within the album. Example: I'm lokking for a specific beach I've been, I've been many times in Greeece, I do remmeber wich holiday trip is realted this pic, but not the exact isald, so searching on the map with all of the pics would be a waste time, If I could search within a specifica album it would be easier to find it

**Comment by qrlt** [comment]
+1 for this feature request

**Comment by MaxenceDC** [comment]
hacky workaround in the meantime : create a dummy user which has only access to said album, and view the map from there.

**Comment by Thornleigh0** [comment]
This one feature would make a huge difference for business use. I would love to host Immich for our office photos when we visit jobsites. To be able to see where a photo was taken from on a map view of the job, along with having a universally accessible album of photos for each job would put Immich in competition with apps like CompanyCam.

**Comment by papuass** [comment]
Looks like this was just implemented in https://github.com/immich-app/immich/releases/tag/v1.133.0
